subsys/bluetooth/mesh/sensor_types.c

```diff
@@ -517,21 +517,21 @@ FORMAT(co2_concentration)     = SCALAR_FORMAT_MAX(2,
 					      HAS_UNDEFINED),
 					      ppm,
 					      SCALAR(1, 0),
-					      65533);
+					      65534);
 FORMAT(noise)			     = SCALAR_FORMAT_MAX(1,
 					       (UNSIGNED |
 					       HAS_HIGHER_THAN |
 					       HAS_UNDEFINED),
 					       db,
 					       SCALAR(1, 0),
-					       253);
+					       254);
```

Comment:
Does this change the behavior in any way? This module is so convoluted that I'm having a hard time figuring out the consequences of changing these for "unrelated" types.

Reply:
Actually, there are no changes in behavior for these modules at all (even unit tests continued to work as usual).
There is a discrepancy in the specification for some characteristics. For example, take a look at noise characteristic: 
valid range 0 - 253 for sensors in decibels
special value 254 for noise with 254 decibels or higher
We implemented handler for valid range separately and handler for special values separately.
I changed it a little bit. 
valid range is 0 - 254 but if there is flag `HAS_HIGHER_THAN` then the top boundary represents all values above the boundary. 
We implemented the first variant because all special values were possible to represent like (type max - 2) but the Voltage sensor broke this representation due to quite original representation in 1\64 V units. Max value is not even close to type max value. 65408 represents 1022.0 Volts and higher.

Reply:
Right. I think everything is good then.